test/framework/kubernetesversions/template.go

```diff
@@ -17,27 +17,33 @@ limitations under the License.
 package kubernetesversions
 
 import (
+	"bytes"
 	_ "embed"
 	"errors"
+	"fmt"
 	"os"
 	"os/exec"
 	"path"
+	"strings"
+	"text/template"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	cabpkv1 "sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1alpha4"
-	kcpv1 "sigs.k8s.io/cluster-api/controlplane/kubeadm/api/v1alpha4"
-	"sigs.k8s.io/cluster-api/test/framework"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"sigs.k8s.io/yaml"
-)
 
-const yamlSeparator = "\n---\n"
+	"sigs.k8s.io/cluster-api/test/framework"
+)
 
 var (
-	//go:embed data/kustomization.yaml
-	kustomizationYamlBytes []byte
+	//go:embed data/debian_injection_script_control_plane.envsubst.sh
+	debianInjectionScriptControlPlaneBytes string
```

Comment:
An alternative to duplicating the script would have been to template the images we pull (that's the only difference between the two scripts). I just thought another layer of templating would be a bad idea.

I'm not sure if there is a good way to auto-discover if the script runs on a control plane or worker node.